docs/02-basic-tutorial.md

```diff
@@ -0,0 +1,363 @@
+# Reflex tutorial
+
+As an introduction to Reflex, we’ll build a simple application showing a list of items. We pick a listing because it’s a very common use case and the simplest example that demonstrates all the major concepts behind Reflex.
+
+## Create a Reflex application
+
+The easiest way to start is using the Reflex CLI. You can install it from npm with
+
+    npm install -g reflex-cli
+
+To create a Reflex application create a new directory and run `reflex init` inside
+
+    mkdir demo
```

Comment:
Can I suggest that you chain the creation and cd into the directory here (i.e. `mkdir demo && cd demo`), so as to avoid any confusion/mistakes - people like to copy & paste and might skim over things...